packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md

```diff
@@ -5,12 +5,18 @@ You may be interested in this document if:
 - you want to create your own implementation of the Request protocol
 - you are curious enough to dive and see what is under the hood of the Request protocol
 
-Prerequisite: Having read the advanced logic specification (see [here](./advanced-logic-specs-0.1.0.md)).
+Prerequisite: Having read the advanced logic specification (see [here](./advanced-logic-specs-0.2.0.md)).
```

Comment:
why did you bump this file? do you plan on bumping all specs?

Reply:
Good one. It was just a mistake.

Reply:
seems like a search/replace mistake :)